### DISCUSSION TITLE
Provide support for events

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [appwrite/appwrite#1381] [Ideas]
## 🚀 Feature

Hi,

There is any plans to support events and event search/analytics on AppWrite?

Thanks.

**Comment by eldadfux** [comment]
Yes we do have this feature on the long term roadmap. I will leave this issue open to share more updates, and to get more community feedback.

**Comment by Meldiron** [comment]
Analytics such as daily users or currently online users are in my opinion must-have for any project, whether it is a mobile or web app.

Then I can see 2 ways a developer may want to go:
- App
- Game

I believe Appwrite can be easily used as a backend for a game to hold leaderboards, events, accounts... These game developers surely want to get features to "monitor" players to see stats such as "when a player left the game", "what level took him the longest", "how many times did average user die on this level".

On the other hand, Appwrite is used by a lot of web developers to create web apps and SAAS. As a web developer, I would love to know what page is visited the most, where a user is clicking the most and if he can find what he is looking for (category on e-shop).

---

With that said, this seems to be really complex problem and will require a lot of work to be perfect.

**Comment by Meldiron** [comment]
How will this integrate into web apps hosted on appwrite with static hosting? (future feature) Would it be able to automatically hook into a website so I as a developer don't even have to write a single line of code? (for basic stuff only, of course)

**Reply by paulocoutinhox** [reply]
Maybe Firebase SDK sources can help:

https://github.com/firebase/firebase-ios-sdk
and
https://github.com/firebase/firebase-android-sdk

**Comment by mbos2** [comment]
Such feature would be huge improvement and would open doors to more appwrite apps in my oppinion.
Even tho such feature can be implemented in any project, it requires a lot of work, and having it in appwrite would definitelly save much of dev time.
